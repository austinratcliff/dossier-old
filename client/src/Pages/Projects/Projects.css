.Projects {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  background-color: white;
  color: #111;
  font-family: 'IBM Plex Mono', monospace;
  height: 100vh;
  width: 100vw;
}

.Projects .header {
  font-size: calc(10px + 5vmin);
  margin: 0;
}

.Projects .home {
  position: absolute;
  left: 0;
  margin-left: 25px;
  cursor: pointer;
}
