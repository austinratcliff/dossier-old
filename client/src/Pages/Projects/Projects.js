import React from 'react'
import './Projects.css'

const Projects = () => {
  return (
    <div className="Projects">
      <h1 className="header">
        <i
          className="fas fa-code"
          title="projects"
        /> Projects
      </h1>
      <i
        className="fas fa-angle-right fa-2x home"
        title="home"
      />
    </div>
  )
}

export default Projects
