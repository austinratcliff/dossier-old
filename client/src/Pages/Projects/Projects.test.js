import React from 'react'
import ReactDOM from 'react-dom'
import ReactTestRenderer from 'react-test-renderer'
import Projects from './Projects'

describe('Projects component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Projects />, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  it('matches snapshot', () => {
    const component = ReactTestRenderer.create(<Projects />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
