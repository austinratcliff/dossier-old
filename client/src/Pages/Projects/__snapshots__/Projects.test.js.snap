// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects component matches snapshot 1`] = `
<div
  className="Projects"
>
  <h1
    className="header"
  >
    <i
      className="fas fa-code"
      title="projects"
    />
     Projects
  </h1>
  <i
    className="fas fa-angle-right fa-2x home"
    title="home"
  />
</div>
`;
